swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala

```diff
@@ -397,6 +397,7 @@ class SwaggerSpec2 extends ScalatraSpec with JsonMatchers {
             (JString(operationId) must_== operation \ "operationId") and
               verifyOperation(operation, swaggerJsonJValue \ "paths" \ path \ method, operationId)
         }
+      case _ => None
```

Comment:
Not sure I understand this one, looking at the data above it should never be neered, right...?

Reply:
The compiler was screaming about the exhaustiveness check. Although I have to keep in mind this is not our code base and I'll keep the changes to the minimum